tasks/task01/Narcissistic_Number.py

```diff
@@ -0,0 +1,16 @@
+for k in range(1, 1000) :
+    if(k>=1 and k<10):
+        digit=k%10
+        if digit**1==k:
+         print ('Narcissistic__Number', k)
```

Comment:
```suggestion
         print (f"Narcissistic__Number {k}")
```
Есть классная штука - [f-string](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python), лучше использовать её
Ошибки тут нет, но так будет лучше